### ISSUE TITLE
Failed to load the preview

### ISSUE THREAD

**Issue by FX7** [Radiokot/photoprism-android-client#88]
**What is wrong?**
When clicking on a thumbnail a loading animation is shown and then the error "Failed to load the preview" is shown.

**To Reproduce**
1. Go to 'select any picture from the starting picture Overview"
2. Click on 'the picture you want to see in fullscreen"

**Expected behavior**
The picture in fullscreen.

**Screenshots**
![Screenshot_20230924-081244_1](https://github.com/Radiokot/photoprism-android-client/assets/5314850/8d3ea9ca-5f82-4d29-935f-7959b0e44890)
![Screenshot_20230924-081519_1](https://github.com/Radiokot/photoprism-android-client/assets/5314850/a499c92d-b916-400f-9a02-58c4a5853b0c)

**Device:**
 - Model: Pixel 6a
 - Android version: 13

**Additional context**
No Errors in the Server Side logs. All requests are answered with http 200. Access to the fullscreen Image from the web Interface works as expected.

**Comment by Radiokot** [comment]
Are the square thumbnails displayed on the main screen?

**Comment by FX7** [comment]
Hello @Radiokot and thank you for the very quick response.

Yes, the square thumbnails on the main screen are displayed.

I noticed the error with the PhotoPrism App v1.17 and updated today to the 1.18 version. But the problem remains.

**Comment by Radiokot** [comment]
I think it can be due to the inability to get the preview of the corresponding size from the server, which I haven't faced before. You said you have server logs – do you see requesting XXX/tile_500, XXX/fit_720, XXX/fit_1280, etc. when running the gallery?

**Comment by FX7** [comment]
First a quick description of my setup:
I have the latest photoprism docker image from photoprism/photoprism:latest behind an apache proxy.

When opening the app I got many log entrys like "GET /api/v1/ ..." with a http 200 code answered in the (apache proxy) log files. As descriped before everythink works fine (small square previews of the starting page are all shown correctly).

Then, wenn clicking on one image I got 3 entrys like "GET /api/v1/t/71866bb4cae0294cde5198c7c7e1184dc1669a09/34mwljxd/fit_3840" and all with http 200.

...

Ok, wait! I solved it! I'm sorry, that I bothered you with this. It was an configuration failure on my server side! Just because the "same" action on the web-interface worked, I thought it was a problem with the app. But "the same" action isnt really the same. Clicking on a square preview image on the web interface doesnt open the original image, it just opened another generated preview. Clicking on the square preview in the app opens the original image.

And thats where my server was missconfigured. I moved the location of the original images, but didnt tell the server the correct new location. So all previously generated previews still work (= square images and clicking on them the in web-interface) but viewing the original image (= clicking on square images in the app) didnt work.

Just solved it by telling the photoprism server the correct new location of the original files.

Thanks again for your quick response and of corse for sthe great work with this app!
